src/app/containers/FrontPageSection/__snapshots__/index.test.jsx.snap

```diff
@@ -153,6 +153,7 @@ exports[`FrontPageSection Container snapshots should render correctly for canoni
         },
       }
     }
+    visuallyHidden={true}
```

Comment:
Sorry, now we have the opersite problem, theyre all true 🙈 Can we get a tests that shows false too please?